### PR TITLE
Nurse alert frontend (Send alert and show alert messages)

### PR DESCRIPTION
**Pull request by Yqiyuan**
# Description

This PR
1. Add send nurse alert panal
2. Show nurse alert messages
3. Connect nurse alert frontend to backend
4. Add stories

https://user-images.githubusercontent.com/12007072/228074511-df01f7c7-6560-4ede-a2f3-7df4741aa9ad.mov

# Test Plan

Please explain how this change will be tested.

- [x] Unit test
- [x] Integration test (including unit tests with supertest)
- [x] Manual test (Please explain below for the test steps)
- [ ] Not Applicable

# Task Link

Please add related jira task links

### CODE REVIEW

File: client/src/components/Alerts/NurseAlertPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  }, [])
+
+  return (
+    <Stack spacing={2}>
+      <Box display={'flex'} width={'100%'}>
+        <SquareButton
+          variant="contained"
+          color={'error'}
+          onClick={async () => {
+            await helpOnClickHandler(ALERTS.NURSE_E_HELP)
```
Comment (jennyzeng) [inline client/src/components/Alerts/NurseAlertPanel.tsx:94]: nit: extract this as a callback function and handle error?

File: client/src/components/Alerts/NurseAlertPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+          }}
+        >
+          <Typography variant={'h5'}>E HELP</Typography>
+        </SquareButton>
+        <SquareButton
+          variant="contained"
+          sx={{
+            'backgroundColor': 'orange',
```
Comment (jennyzeng) [inline client/src/components/Alerts/NurseAlertPanel.tsx:102]: maybe use a theme color by doing 
```
'backgroundColor': (theme) => theme.palette.warning,
```

File: client/src/components/Alerts/NurseAlertPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+            value={selectedPatientId}
+            onChange={(event) => {
+              setSelectedPatientId(event.target.value)
+            }}
+          >
+            {selectPatientMenuItems}
+          </Select>
+        </Box>
+
+        <Box display={'flex'} alignItems={'center'}>
```
Comment (jennyzeng) [inline client/src/components/Alerts/NurseAlertPanel.tsx:142]: nit: remove unnecessary curly braces

File: client/src/components/Alerts/NurseAlertPanel.tsx [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+  }
+}) => {
+  await request(`/api/channels/${channelId}/alerts`, {
+    method: 'POST',
+    body: JSON.stringify({
+      content: type,
+      detailedContent,
+    }),
+  })
+}
```
Comment (jennyzeng) [inline client/src/components/Alerts/NurseAlertPanel.tsx:197]: nit: put this in the `requests` directory

File: client/src/components/ChatRoomPageContent.tsx [hunk 5]
```diff
@@ -32,23 +33,31 @@ export function ChatRoomPageContent({
 
   openAlertPanel,
 }: ChatRoomPageContentParams) {
+  const role = localStorage.getItem('role')
```
Comment (jennyzeng) [inline client/src/components/ChatRoomPageContent.tsx:36]: `useLocalStorage`

File: client/src/components/ChatRoomPageContent.tsx [hunk 6]
```diff
@@ -32,23 +33,31 @@ export function ChatRoomPageContent({
...
+  const isNurse = role === 'Nurse'
```
Comment (jennyzeng) [inline client/src/components/ChatRoomPageContent.tsx:37]: Use `ROLE.Nurse`

**Review by jennyzeng — APPROVED** [review]
just multiple nit comments

**Review by bettsongy — APPROVED** [review]

**Comment by Yqiyuan** [comment]
> just multiple nit comments

Thank you for reviewing my code. I will fix them.

File: client/src/components/Alerts/NurseAlertPanel.tsx (hunk 2 above)
Comment (Yqiyuan) [reply client/src/components/Alerts/NurseAlertPanel.tsx:105]: theme.palette.warning.light is not orange as the default color. It is pink.

**Pull request merged**